src/app/preferences/general-preferences/general-preferences.service.ts

```diff
@@ -0,0 +1,17 @@
+namespace Core {
+    export class generalPreferencesService {
```

Comment:
It's not the class name that I request to change to lower capital. It's the service reference name.

Reply:
I think this is resolved.